### ISSUE TITLE
The use of this on my website

### ISSUE THREAD

**Issue by AnanthVivekanand** [C0nw0nk/CoinHive#5]
I was planning to use this on my webpage until I noticed the license said copyright. So may I directly implement your example onto my webpage or do I have to write my own code to do this?

**Comment by C0nw0nk** [comment]
Yeah the copyright is just the default github copyright license, Even repositorys without a license shown or displayed fall under the default copyright law. (The same as what mine shows)

Feel free to use it i will change the license when i get time to one of the ones everyone can use. In the meantime if you do receive any copyright complaints and notices from copyright trolls, they are not from me and you can link to this page and issue request to show that they are not from me since I do not care who runs this script. That will allow you to shut them up.
